### PR TITLE
Methods mset, mget and madd have been marked as deprecated

### PR DESCRIPTION
**Pull request by githubjeka**
http://stackoverflow.com/questions/34008096/why-yii2-uses-abbreviated-methods-names-in-yii-caching-cache

### CODE REVIEW

File: framework/caching/Cache.php [hunk 1]
```diff
@@ -207,6 +224,7 @@ public function set($key, $value, $duration = 0, $dependency = null)
      * If the cache already contains such a key, the existing value and
      * expiration time will be replaced with the new ones, respectively.
      *
+     * @deprecated This method is alias as [[multiSet()]] and will be removed in 2.1.0.     *
```
Comment (SilverFire) [inline framework/caching/Cache.php]: Flying star :)

Comment (githubjeka) [inline framework/caching/Cache.php]: I'm so sorry.

**Comment by samdark** [comment]
Looks good. Thanks @githubjeka!

**Comment by githubjeka** [comment]
Спасибо (+1) для @githubjeka принимается! Теперь его карма +3.

**Pull request merged**